src/target/fhe/utils.rs

```diff
@@ -0,0 +1,46 @@
+//! Utility functions to write compiler output to FHE
```

Comment:
Can you leave a TODO to parameterize these functions so they can be shared between the ABY and SEAL backends?